Edit Embedded Wallet Android SDK builder content (#2273)
MetaMask/metamask-docs: src/pages/quickstart/builder/embedded-wallets/android/stepContent/blockchainCalls.mdx

### Make blockchain calls

After the user signs in, the SDK provides a user-scoped private key and session ID.

> Use the private key to interact with chains such as [Ethereum](https://ethereum.org/) to make
> blockchain calls, like fetching a user's account balance, sign or send
> transactions, read and write to smart contracts, on so on.
> 
> We added a basic EVM chain based example here for your reference.

Reviewer: ```suggestion
Use the private key to interact with chains such as [Ethereum](https://ethereum.org/) to make
blockchain calls, like fetching a user's account balance, signing or sending
transactions, and reading or writing to smart contracts.

This quickstart includes a basic EVM-based example for your reference.
```

:::info Connect any Blockchain

> Refer to the [Blockchain Documentation](/connect-blockchain/) to learn more about making calls on

Reviewer: ```suggestion
Refer to the [blockchain documentation](/embedded-wallets/connect-blockchain/) to learn more about making calls on
```
